### PR TITLE
Allow local and remote executable to be specified

### PR DESCRIPTION
**Pull request by frankcrawford**
This allows safekeep to run a newer version of rdiff-backup against one host and still run a different (older) one against a different host.

### CODE REVIEW

File: doc/safekeep.backup.txt [hunk 1]
```diff
@@ -190,6 +190,14 @@ are separated by '/', attributes are introduced by '@':
...
        different options will be made available for other backends.
 
+/backup/options/rdiff-backup/@exec_local::
+       Executable to be run on the local host.
+       Optional, defaults to 'rdiff-backup'
+
+/backup/options/rdiff-backup/@exec_remote::
+       Executable to be run on the remote host.
+       Optional, defaults to 'rdiff-backup'
+
```
Comment (HRio) [inline doc/safekeep.backup.txt:200]: how are these options used? Would be good if an example of backup options could added in the examples?

```
 <options>
  <rdiff-backup exec_local="rdiff-backup-1.2.8"/>
 </options>
```

but would be nice to have it as an example.

**Comment by HRio** [comment]
looks good and works for me.

File: doc/safekeep.backup.txt (hunk 1 above)
Comment (frankcrawford) [reply doc/safekeep.backup.txt:200]: Yes, that is how it is used, although I'll see if I can add an example within the file.

**Comment by HRio** [comment]
@frankcrawford rdiff-backup 1.2.8 is now back in Alpine master (with me as maintainer). I plan to keep it in testing for a week or so and later move it to community in time for the v3.13 release of Alpine. Would be nice to also be able to ship a new version of safekeep with this PR in it as well.

**Comment by frankcrawford** [comment]
@HRio I plan to merge it in the next day or so and do a new release straight after, so you should have it available in the next week.

File: doc/safekeep.backup.txt (hunk 1 above)
Comment (HRio) [reply doc/safekeep.backup.txt:200]: would be nice if an example could be added before release as the right syntax is crucial to have correct in order to not start the backup with the wrong version.

**Comment by frankcrawford** [comment]
@HRio I've added in an example in both the documentation (safekeep.backup.txt) and the sample script, although in both cases commented out, in case someone uses them verbatim.
See what you think, meanwhile I'll start the release process this weekend.

**Comment by HRio** [comment]
> @HRio I've added in an example in both the documentation (safekeep.backup.txt) and the sample script, although in both cases commented out, in case someone uses them verbatim.
> See what you think, meanwhile I'll start the release process this weekend.

looks very good thank you!

**Pull request merged**